File: TimeToCode/app/src/main/java/com/kmu/timetocode/Announce.java
package com.kmu.timetocode;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageButton;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

public class Announce extends Fragment {
    public static Announce newInstance(){
        return new Announce();
    }
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        ViewGroup announce = (ViewGroup) inflater.inflate(R.layout.fragment_announce, container, false);
        ImageButton backButton = announce.findViewById(R.id.backCertification);
        ImageButton AnnouncePageButton = announce.findViewById(R.id.announcePageButton);
        backButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                ((NavActivity)getActivity()).replaceFragment(MainFragment.newInstance());
            }
        });
        AnnouncePageButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                ((NavActivity)getActivity()).replaceFragment(AnnouncePage.newInstance());
            }
        });
        return announce;
    }
}

File: TimeToCode/app/src/main/java/com/kmu/timetocode/Announce.kt
package com.kmu.timetocode

import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import android.widget.ImageButton
import androidx.fragment.app.Fragment

class Announce : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val announce = inflater.inflate(R.layout.fragment_announce, container, false) as ViewGroup
        val backButton = announce.findViewById<ImageButton>(R.id.backCertification)
        val AnnouncePageButton = announce.findViewById<ImageButton>(R.id.announcePageButton)
        backButton.setOnClickListener { (activity as NavActivity?)!!.replaceFragment(MainFragment.newInstance()) }
        AnnouncePageButton.setOnClickListener {
            (activity as NavActivity?)!!.replaceFragment(
                AnnouncePage.newInstance()
            )
        }
        return announce
    }

    companion object {
        @JvmStatic
        fun newInstance(): Announce {
            return Announce()
        }
    }
}